clients/web/src/components/app/spaces/upload-file.tsx

```
@@ -1,16 +1,46 @@
+  const handleClick = async () => {
+    const fileInput = document.createElement("input");
+    fileInput.type = "file";
+    fileInput.onchange = async (e) => {
+      const file = (e.target as HTMLInputElement).files?.[0];
+      if (file) {
+        const response = await handleUpload(spaceId, file);
+        if (response.error) {
+          toast.error(response.error.message);
+        }
+        if (response.data) {
+          if (response.data.data && response.data.data.message) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)